PurchasesTests/TestHelpers/ObjCThrowExceptionMatcher.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ObjCThrowExceptionMatcher.swift
```

Comment:
Maybe we should add a note here, pointing to [the line in Nimble that specifies that their matcher doesn't work with SPM](https://github.com/Quick/Nimble/blob/main/Sources/Nimble/Matchers/RaisesException.swift#L1), so that future devs understand why we have our own matcher specifically for this